src/index.ts

```diff
@@ -0,0 +1,8 @@
+function helloworld (msg = 'Hello World!'): string {
```

Comment:
The ideia in the `index` is to import the `helloWorld` function from `modules` folder so then you can call `helloWorld()`

Reply:
Ok, just fixed it.